docs/docs/casper/deployment.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Deployment
+
+slug: /casper/deployment
+---
+
+This section provides detailed instructions on how to deploy the Invariant protocol on Casper, including prerequisites.
+
+## Prerequisites
+
+- Rust & Cargo ([rustup](https://www.rust-lang.org/tools/install))
+- wasm-strip ([wabt](https://github.com/WebAssembly/wabt))
+- cargo-odra ([cargo-odra](https://github.com/odradev/cargo-odra))
+- Download & Build ([Installation](http://localhost:3000/docs/casper/installation))
+- casper-client ([casper-client](https://github.com/casper-ecosystem/casper-client-rs))
+
+#### Rust & Cargo
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+#### wasm-strip
+
+```bash
+sudo apt install wabt
+```
+
+#### Add WASM
+
+```bash
+rustup target add wasm32-unknown-unknown
+```
+
+#### cargo-odra
+
+```bash
+cargo install cargo-odra --locked
+```
+
+#### Download & Build
+
+```bash
+git clone git@github.com:invariant-labs/protocol-cspr.git
+cargo odra build -b casper
```

Comment:
Is there a release version of the deployment? If so, it should definitely be included here